nautobot/core/tests/integration/test_plugin_home.py

```diff
@@ -0,0 +1,123 @@
+from django.test.utils import override_settings
+from django.conf import settings
+from unittest import skipIf
+
+from nautobot.circuits.models import Circuit, Provider
+from nautobot.dcim.models import PowerFeed, PowerPanel, Site
+from nautobot.tenancy.models import Tenant
+from nautobot.utilities.testing.integration import SeleniumTestCase
+
+from dummy_plugin.models import DummyModel
+from dummy_plugin.homepage import get_dummy_data
+
+
+@skipIf(
+    "dummy_plugin" not in settings.PLUGINS,
+    "dummy_plugin not in settings.PLUGINS",
+)
+class PluginHomeTestCase(SeleniumTestCase):
+    """Integration test the plugin homepage."""
+
+    fixtures = ["user-data.json"]  # bob/bob
+    layout = {
+        "Organization": {
+            "Sites": {"model": Site, "permission": "dcim.view_site"},
+            "Dummy Models": {"model": DummyModel, "permission": "dummy_plugin.view_dummymodel"},
+            "Tenant": {"model": Tenant, "permission": "tenancy.view_tenant"},
+        },
+        "Dummy Plugin": {
+            "Dummy Models": {"model": DummyModel, "permission": "dummy_plugin.view_dummymodel"},
+        },
+        "Power": {
+            "Power Feeds": {"model": PowerFeed, "permission": "dcim.view_powerfeed"},
+            "Power Panel": {"model": PowerPanel, "permission": "dcim.view_powerpanel"},
+        },
+        "Circuits": {
+            "Providers": {"model": Provider, "permission": "circuits.view_provider"},
+            "Circuits": {"model": Circuit, "permission": "circuits.view_circuit"},
+        },
+    }
+
+    custom_panel_dummymodel = {
+        "name": "Custom Dummy Plugin",
+        "items": [
+            "Example 1",
+            "Example 2",
+            "Example 3",
+        ],
+    }
+
+    def setUp(self):
+        super().setUp()
+        self.login(self.user.username, self.password)
+
+    def tearDown(self):
+        self.logout()
+        super().tearDown()
+
+    def test_homepage_render(self):
+        """
+        Render homepage with app defined objects.
+        """
+        # Set test user to admin
+        self.user.is_superuser = True
+        self.user.save()
+
+        self.load_page(self.live_server_url)
+
+        columns_html = self.selenium.find_elements_by_class_name("homepage_column")
+        for panel_name, panel_details in self.layout.items():
+            columns_html[0].find_element_by_xpath(f".//strong[contains(text(), '{panel_name}')]")
```

Comment:
Why not an exact match instead of `contains()`?

```suggestion
            columns_html[0].find_element_by_xpath(f".//strong[text()='{panel_name}']")
```

Same comment throughout - maybe retroactively applicable to the `test_home.py` tests added in your previous PR as well.

Reply:
Mind making this same change on `test_home.py`?